src/sentry/auth/authenticators/u2f.py

```diff
@@ -69,10 +78,13 @@ def start_enrollment(self, user, is_webauthn_register_ff):
         if is_webauthn_register_ff:
             credentials = []
             for registeredKey in self.get_u2f_devices():
-                c = self._create_credential_object(registeredKey)
-                credentials.append(c)
+                if not type(registeredKey) == AuthenticatorData:
```

Comment:
if the order doesn't matter when checking a boolean, it's best to use a true value in an if statement since its a little more readable.

Reply:
(remove the not and swap the else block with the if's)